tests/test_hear_server_activate.py

```
@@ -83,7 +83,7 @@ def handle(self):
 				a request. In this case, it is just a test fixture and does not
 				perform any actions.
 				"""
-				pass  # skipcq: PTC-W0107 -- Handler logic is not the focus here
+				pass  # skipcq: PYL-W0107 -- Handler logic is not the focus here
```

Ruff (0.15.21)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)